astropy/coordinates/baseframe.py

```diff
@@ -1162,6 +1162,10 @@ def transform_to(self, new_frame):
             if new_frame is self.__class__:
                 # no special transform needed, but should update frame info
                 return new_frame.realize_frame(self.data)
+            elif isinstance(new_frame, str):
+                raise ConvertError("Could not find frame with name " +
+                                   "'{}' ".format(new_frame) +
+                                   "in the frame transformation graph")
```

Comment:
full stop at the end?! :laughing:

Reply:
I think @MSeifert04 and I also had a looong discussion on whether to capitalize the error message or not. It was a long time ago and I don't remember the conclusion anymore. :sweat_smile:

Reply:
it's  uncapitalized

Reply:
(as I remember this was a discussion with @taldcroft, but now I can't find the reference to it, bottom line that the error message starts with the errors name and not as a new sentence.)